pkg/security/util.go

```diff
@@ -2,6 +2,7 @@ package security
 
 const (
 	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
+	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
```

Comment:
> On second thought... maybe what you have will work. For the case I just described, you could use the first group as the allocated one, and put the allowed supplemental groups for shared storage after that in the list.

do we agree then, that nothing needs to change here?  I think the only open item from your comments is whether or not the admission strategy allows the first GID in the annotation to be part of the allowed supplemental groups when validating or if it is reserved for fsgroup id and if you don't mind sharing you'd set up the annotation like `1,1-10`.